wa/framework/version.py

```diff
@@ -21,7 +21,7 @@
 
 VersionTuple = namedtuple('Version', ['major', 'minor', 'revision'])
 
-version = VersionTuple(3, 1, 1)
+version = VersionTuple(3, 1, 'dev2')
```

Comment:
Shouldn't this be `VersionTuple(3, 1, 'dev2')`?

Reply:
Good point, updated to add a development field to the version.